src/client/dfuse/dfuse_core.c

```diff
@@ -517,6 +670,35 @@ dfuse_cont_open(struct dfuse_projection_info *fs_handle, struct dfuse_pool *dfp,
 						strerror(rc));
 				D_GOTO(err_close, rc);
 			}
+
+			if (fs_handle->dpi_info->di_caching) {
+				rc = dfuse_cont_get_cache(dfc);
+				if (rc == ENODATA) {
+					/* If there is no container specific
+					 * attributes then use defaults
+					 */
+					DFUSE_TRA_INFO(dfc,
+						       "Using default caching values");
+					dfc->dfc_attr_timeout = 1;
+					dfc->dfc_dentry_timeout = 1;
+					dfc->dfc_dentry_dir_timeout = 5;
+					dfc->dfc_ndentry_timeout = 5;
```

Comment:
A more cautious approach would to to set this to 0, at least until we better understand not to handle negative dentry caching on unlink.

Reply:
I've set it to 1